### PR TITLE
Gameini updates.

### PR DESCRIPTION
**Pull request by Linktothepast**
Fixes issue 8637 by replacing SkipIdle=0 with SyncOnSkipIdle = False.
Updates gameinis for Resident Evil Archives: Resident Evil Zero, Pokemon
Snap and fixes a mistake in Baldur's Gate: Dark Alliance.

### CODE REVIEW

**Comment by degasus** [comment]
As far as I know at least "Need for Speed: Hot Pursuit 2" also should have "SyncOnSkipIdle = False"

**Comment by JosJuice** [comment]
Can you also add FastDiscSpeed for these two games? https://code.google.com/p/dolphin-emu/issues/detail?id=8650

File: Data/Sys/GameSettings/G8W.ini [hunk 1]
```diff
@@ -19,9 +19,4 @@ EmulationIssues = Idle skipping slows down the game.
 # Add action replay cheats here.
 
 [Video]
```
Comment (JosJuice) [inline Data/Sys/GameSettings/G8W.ini]: The [Video] sections should be deleted if they are empty.

**Comment by Linktothepast** [comment]
Degasus can you give more details about that? I can't find a speedup with it for need for speed hp2.
JosJuice, yeah i can do that.

**Comment by degasus** [comment]
@Linktothepast It's a variable FPS game, so the FPS won't be affected, but the VPS. It gives the emulator a way to still get fullspeed (VPS=60) through not reaching the max FPS. Sounds strange, but is accurate as also the GC likely only get 30 FPS in this game.

**Comment by Linktothepast** [comment]
I tried again but i see no difference in the vps either here. The game has virtual xfb enabled by default without which it gives crazy framerates btw.
There is also the star wars rebel strike that seems to get a speedup from this should i enable it? It isn't necessary for the game but it gives a speedup at certain situations.

**Comment by degasus** [comment]
Aren't "crazy framerates" good? But I'm fine as it's now.

I guess almost all games will get a good speedup with this option, but it's also very often unsafe to do so. So I wouldn't disable just by luck :/

**Comment by Linktothepast** [comment]
"Aren't "crazy framerates" good?"
They are not if you are using vsync, games will crawl.

**Comment by Stevoisiak** [comment]
In the next INI update, can you include that Sam & Max: Season One (R3XE6U) does not work if External Frame Buffer is disabled? It seems to work fine with either Virtual or Real frame buffer enabled.

**Comment by Linktothepast** [comment]
Yes sure thing.

**Pull request merged**